deps/spidershim/src/rootstore.h

```diff
@@ -119,10 +119,10 @@ class RootStore {
   }
 
  private:
-  JS::Rooted<ValueVector> values;
-  JS::Rooted<ScriptVector> scripts;
+  JS::PersistentRooted<ValueVector> values;
+  JS::PersistentRooted<ScriptVector> scripts;
   std::vector<Script*> scriptObjects;
-  JS::Rooted<SymbolVector> symbols;
+  JS::PersistentRooted<SymbolVector> symbols;
```

Comment:
I'd like to understand the details.  This can probably be forked off into its own issue.